### DISCUSSION TITLE
When try to run `maturin build --release --zig ...` the command fails complaining there are missing libraries

### DISCUSSION THREAD

**Discussion by fbrv** [PyO3/maturin#1590] [Q&A]
Hello and thanks for developing this fantastic project.
I started using this afternoon maturin/pyo3 and I want to create a binding on top of a library that uses gstreamer.  
If I run the command `maturin develop` I'm able to create a package, import into python and use it but when I run
```sh
maturin build --release --zig --target x86_64-unknown-linux-gnu --features abi3
```
To create a release in order to make it available to other users, the command return an error, complaining:  
```
  = note: ld.lld: error: unable to find library -lgstapp-1.0
          ld.lld: error: unable to find library -lgstbase-1.0
          ld.lld: error: unable to find library -lgstreamer-1.0
          ld.lld: error: unable to find library -lgobject-2.0
          ld.lld: error: unable to find library -lglib-2.0
          ld.lld: error: unable to find library -lgio-2.0
```
My best guess is that, somehow, maturin/zig is not able to use the static libraries, any idea how I could fix this problem?

**Comment by konstin** [comment]
could you try if it works without the `--target` flag? iirc if you set a target zig believes you're cross compiling and doesn't use the default search paths
